tests/vmi_multus_test.go

```diff
@@ -646,10 +653,10 @@ var _ = Describe("SRIOV", func() {
 
 			_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
 			Expect(err).ToNot(HaveOccurred())
-			tests.WaitUntilVMIReady(vmi, tests.LoggedInFedoraExpecter)
 
 			// Need to wait for cloud init to finish and start the agent inside the vmi.
 			tests.WaitAgentConnected(virtClient, vmi)
+			tests.WaitUntilVMIReady(vmi, tests.LoggedInFedoraExpecter)
```

Comment:
Why did you need to move this?

Reply:
@fedepaol I had this change split into a separate commit that explains the reason in a commit message. Strictly speaking, this commit can be merged separately, I just needed it to consistently pass the suite locally: https://github.com/kubevirt/kubevirt/pull/2692/commits/f96f646edd8a75ad6ceb2b88d952fc7c9857a5fd Would you like me to split it out?

Reply:
@booxter no it's fine, sorry I did not read the commit message. All clear now.

Reply:
Is it worth checking "WaitUntilVMIReady"?
If the agent is connected, then the vm must be up.

Reply:
@mmirecki AFAIU we use the expecter to log in for next commands to succeed. Otherwise the next expecter checks will try to type in commands into `getty` login session.